MAINTAINERS

```diff
@@ -1 +1,10 @@
-Jimmi Dyson <jimmidyson@gmail.com> (@jimmidyson)
+Jimmi Dyson           <jimmidyson@gmail.com>        ( @jimmidyson )
+Hardy Ferentschik                                   ( @hferentschik )
```

Comment:
@gbraad :smile: , he only told me to remove it otherwise it was there previously :)